src/styles/annotator/highlights.scss

```diff
@@ -1,35 +1,55 @@
 :root {
   // Default highlight styling configuration
   --hypothesis-highlight-color: rgba(255, 255, 60, 0.4);
+  --hypothesis-highlight-secondColor: rgba(206, 206, 60, 0.4);
+  --hypothesis-highlight-thirdColor: transparent;
+
   --hypothesis-highlight-focused-color: rgba(156, 230, 255, 0.5);
-  --hypothesis-highlight-blend-mode: normal;
-  --hypothesis-highlight-decoration: none;
   --hypothesis-highlight-text-color: inherit;
 
-  --hypothesis-highlight-second-color: rgba(206, 206, 60, 0.4);
-  --hypothesis-highlight-third-color: transparent;
+  --hypothesis-color-cyan: #cffafe;
```

Comment:
Both the text color and this "cyan" are in flux at the moment. They are only used by this clustered-highlights feature.